src/services/credentialEvents/companyDetailsV1.ts

```diff
@@ -69,14 +69,21 @@ export default class CompanyDetailsV1Handler implements CredentialEventHandler<'
     const [{ pin_attempt_count: pinAttemptCount }] = await this.db.increment('connection', 'pin_attempt_count', {
       id: connection.id,
     })
-    if (pinAttemptCount > this.env.get('INVITATION_PIN_ATTEMPT_LIMIT')) {
+    if (pinAttemptCount >= this.env.get('INVITATION_PIN_ATTEMPT_LIMIT')) {
       this.logger.warn(
         { connectionId: connection.id },
         'PIN verification attempt count exceeded for connection %s',
         connection.id
       )
+      const problemReportPin: { message: string; pinTries: number } = {
+        message: `PIN verification attempt count exceeded for connection ${connection.id}`,
+        pinTries: 5 - pinAttemptCount,
+      }
+      this.logger.warn(pinAttemptCount)
       await this.db.update('connection_invite', { connection_id: connection.id }, { validity: 'too_many_attempts' })
       await this.db.update('connection', { id: connection.id }, { pin_attempt_count: 0 }) // reset so if a new pin is sent they can try again
+      await this.cloudagent.sendProblemReport(credential.id, JSON.stringify(problemReportPin))
```

Comment:
I know it's not part of the ticket, but is it safer for the PIN originator to be informed after every invalid entry? This prevents someone on the PIN entry side from hacking the system to accept more than the 5 attempts

Reply:
you can't hack it from the pin side. if Bob creates an invite Bob himself keeps a track of the number of wrong pin attempt tries